docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -682,6 +682,7 @@ How to **investigate**:
   - Temporarily scale up queriers to try to stop the bleed
   - Check if a specific tenant is running heavy queries
     - Run `sum by (user) (cortex_query_scheduler_queue_length{namespace="<namespace>"}) > 0` to find tenants with enqueued queries
+    - If remote ruler evaluation is enabled, make sure you understand which one of the read paths (user or ruler queries?) is being affected, run `sum by (user, job) (cortex_query_scheduler_queue_length{namespace="<namespace>"}) > 0` and check the `job` label.
```

Comment:
That should also be on the alert message https://github.com/grafana/mimir/blob/c294a5a5a7c2d043159b49ed5ba593bbc0eb723a/operations/mimir-mixin-compiled/alerts.yaml#L78

```suggestion
    - If remote ruler evaluation is enabled, make sure you understand which one of the read paths (user or ruler queries?) is being affected - check the alert message.
```